Sources/StreamChat/Utils/Logger/Logger.swift

```diff
@@ -234,11 +234,24 @@ public class Logger {
     ///
     /// - Parameters:
     ///   - message: A custom message to log if `condition` is evaluated to false.
+    @available(*, unavailable, renamed: "assertionFailure")
     public func assertationFailure(
         _ message: @autoclosure () -> Any,
         functionName: StaticString = #function,
         fileName: StaticString = #file,
         lineNumber: UInt = #line
+    ) { }
```

Comment:
This shouldn't be empty but it should rather call the new implementation.

Reply:
Also, can you please move this method to `Deprecations.swift`?

Reply:
I changed the `unavailable` mark to `deprecated` and moved the whole method to `Deprecations.swift`